src/ADPCMSoundDecoder.js

```diff
@@ -10,6 +9,9 @@ const log = require('./log');
  * https://github.com/LLK/scratch-flash/blob/master/src/sound/WAVFile.as
  */
 class ADPCMSoundDecoder {
+    constructor (context) {
+        this.context = context;
```

Comment:
@ericrosenbaum this is minor thing, but the word `context` is a fairly overloaded concept in javascript. Sometimes `context` is used as an alias for `this` (an idiom we usually avoid) and it also appears when talking about canvas drawing contexts. For someone coming from outside of an audio world they probably will not know this refers to the audio context. I think for clarity we might want to use `audioContext`.